doc/7/getting-started/.react/standalone/src/App.js

```diff
@@ -0,0 +1,155 @@
+/* snippet:start:1 */
+import React from 'react';
+import kuzzle from "./services/kuzzle";
+import './App.css';
+/* snippet:end */
+/* snippet:start:8 */
+const Message = function (props) {
+  const { message, username } = props;
+  return (
+    <div className={(message.username === username ? 'fromMe' : 'fromOthers') + ' messages'}>
+      <span> User: <b>{message.username}</b>  (</span>
+      <span> {new Date(message.createdAt).toLocaleString().split("GMT")[0]} )</span>
+      <p> {message.value} </p>
+    </div>
+  );
+}
+/* snippet:end */
+/* snippet:start:2 */
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      message: "",
+      messages: [],
+      validate: false,
+    };
+  }
+/* snippet:end */
+  /* snippet:start:3 */
+  valid = async () => {
+    await kuzzle.connect();
+    if (!await kuzzle.index.exists("chat")) {
+      await kuzzle.index.create("chat");
+      await kuzzle.collection.create("chat", "messages");
+    }
+    await this.fetchMessages();
+    await this.subscribeMessages();
+    this.setState({ validate: true });
+  }
+  /* snippet:end */
+  /* snippet:start:10 */
+  subscribeMessages = async () => {
+    await kuzzle.realtime.subscribe(
```

Comment:
Useless `await`, you should directly returns the promise and so `async` is also useless.

Declaring an `async` method here will create a new promise to wrap the result of this method which is `undefined`. 
So you will end up with 2 promises on the event loop instead of one